src/test/common/steps/cookie-steps.js

```diff
@@ -13,6 +21,23 @@ Then(/^all page content is present on the cookies page$/, async function () {
   await checkAllCookiePageContentIsPresentAndCorrect()
 })
 
+Then(/^no back link is shown$/, async function () {
+  try {
+    console.log('Timeout error for getting back link expected')
+    await pages.cookies.getBackLink()
+    assert.fail('Backlink found on page but was not expected')
+  } catch (error) {
+    // no nothing, we expect there not to be a back link.
```

Comment:
**Do** nothing? Also being picky, but `we expect there to be no back link` is better.

Reply:
Updated text.

Reply:
Changed.